src/main/resources/common-services/CDAP/6.0.0/configuration/cdap-site.xml

```diff
@@ -3239,7 +3438,7 @@
     <value></value>
     <description>
       JAAS LoginModule implementation to use when
-      co.cask.security.server.JAASAuthenticationHandler is configured for
+      io.cdap.cdap.security.server.JAASAuthenticationHandler is configured for
```

Comment:
this looks like a mistake (extra cdap not present [here](https://github.com/cdapio/cdap/blob/release/6.0/cdap-common/src/main/resources/cdap-default.xml#L3507) )

Reply:
The class is actually io.cdap.cdap, https://github.com/cdapio/cdap/blob/release/6.0/cdap-security/src/main/java/io/cdap/cdap/security/server/JAASAuthenticationHandler.java, we should fix the cdap-default.xml